packages/flame/lib/src/experimental/fixed_size_viewport.dart

```diff
@@ -26,13 +26,12 @@ class FixedSizeViewport extends Viewport {
 
   @override
   bool containsLocalPoint(Vector2 point) {
-    return point.x.abs() <= size.x / 2 && point.y.abs() <= size.y / 2;
+    final x = point.x, y = point.y;
+    return x >= 0 && x <= size.x && y >= 0 && y <= size.y;
```

Comment:
can we just use https://github.com/flame-engine/flame/blob/main/packages/flame/lib/src/extensions/rect.dart#L28

Reply:
That would require creating a temporary `Rect` object, and then converting a point into an `Offset`. Plus, the Rect's right and bottom edges are exclusive, so that would be a slightly different behavior.